featurehub_config_test.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+from featurehub_config import FeatureHubConfig
+
+
+class FeatureHubConfigTest(unittest.TestCase):
+
+    edge_url = 'http://localhost'
+    client_eval_key = 'default/29517115-da60-4b64-99db-9da017561edd/7NZq23UuFEdmpcwxx7lkKSH0QH1EIS*dmGCHaOErbXyMLfzHCe0'
+    server_eval_key = 'default/29517115-da60-4b64-99db-9da017561edd/a2eK0IJEeXdBrgiv3sLKaaxEZDy6uJ55F85RhIdK'
+    another_client_key = 'default/29517115-da60-4b64-99db-9da017561edd/a2eK0IJEeXdBrgiv3sLKaaxEZDy6uJ55F85RhIdK*dmGCHaOErbXyMLfzHCe0'
+
+    def test_featurehub_config_constructor(self):
+        config = FeatureHubConfig(self.edge_url, self.client_eval_key)
+        self.assertEqual(config._api_keys, ['default/29517115-da60-4b64-99db-9da017561edd/7NZq23UuFEdmpcwxx7lkKSH0QH1EIS*dmGCHaOErbXyMLfzHCe0'])
+        self.assertEqual(config._client_eval, True)
+        self.assertEqual(config._edge_url, 'http://localhost/')
+        self.assertIsNotNone(config._repository)
+
+    def test_featurehub_config_multi_api_keys(self):
+        config = FeatureHubConfig(self.edge_url, self.client_eval_key, self.another_client_key)
+        self.assertEqual(config._api_keys, ['default/29517115-da60-4b64-99db-9da017561edd/7NZq23UuFEdmpcwxx7lkKSH0QH1EIS*dmGCHaOErbXyMLfzHCe0',
```

Comment:
when you use these big long keys, its hard to see if they are importantly  long and this detailed. Like is there a significance to them? I would just use shorter keys that aren't valid for the server but you don't really care about that,

Reply:
yes, I keep them realistic, because it's useful knowing what the data looks like